packages/blocks/src/api/registration.js

```diff
@@ -271,103 +264,15 @@ export function registerBlockType( blockNameOrMetadata, settings ) {
 		usesContext: [],
 		supports: {},
 		styles: [],
+		variations: [],
```

Comment:
Block `variations`, similar to `styles`, default to an empty array on the server:

https://github.com/WordPress/wordpress-develop/blob/0bf7d5ea0dd8761ec3026a01aaa27646db8bd7a4/src/wp-includes/class-wp-block-type.php#L102-L108

Let's align how it is handled.